wallet/src/de/schildbach/wallet/ui/dashpay/NotificationsAdapter.kt

```diff
@@ -0,0 +1,259 @@
+package de.schildbach.wallet.ui.dashpay
+
+import android.text.format.DateUtils
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.ImageView
+import android.widget.TextView
+import androidx.recyclerview.widget.RecyclerView
+import com.bumptech.glide.Glide
+import de.schildbach.wallet.data.DashPayContactRequest
+import de.schildbach.wallet.data.UsernameSearchResult
+import de.schildbach.wallet.ui.UserAvatarPlaceholderDrawable
+import de.schildbach.wallet_test.R
+import kotlinx.android.synthetic.main.contact_request_row.view.*
+import org.bitcoinj.core.Utils
```

Comment:
It looks like this import can be removed.

Reply:
removed